public/app/plugins/panel/graph/Legend/LegendSeriesItem.tsx

```diff
@@ -136,6 +136,8 @@ class LegendSeriesLabel extends PureComponent<LegendSeriesLabelProps & LegendSer
   render() {
     const { label, color, yaxis } = this.props;
     const { onColorChange, onToggleAxis } = this.props;
+    const onLabelClick = this.props.onLabelClick ? this.props.onLabelClick : () => {};
```

Comment:
nit: maybe create a private function that does this check instead of needing to "potentially" create a no-op function on each rendering.